src/Bicep.LangServer/Handlers/BicepDeploymentStartCommandHandler.cs

```diff
@@ -63,6 +78,36 @@ public override async Task<BicepDeploymentStartResponse> Handle(BicepDeploymentS
             var credential = new CredentialFromTokenAndTimeStamp(request.token, request.expiresOnTimestamp);
             var armClient = new ArmClient(credential, default, options);
 
+            string? bicepparamJsonOutput = null;
+
+            if(request.parametersFilePath.EndsWith(".bicepparam"))
+            {
+                //params file validation 
+                if(request.parametersFileUpdateOption != ParametersFileUpdateOption.None)
```

Comment:
[nit] The usual convention in C# is `if (` rather than `if(`. You could probably run the "Format Document" VSCode command (Ctrl+K+D in VS) to auto-fix the formatting.